ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml

```diff
@@ -73,6 +77,19 @@ tests:
     pre-commit run --all-files --show-diff-on-failure --verbose
   container:
     from: openstack-operator-ci-pre-commit
+- as: nova-operator-build-deploy-tempest
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h20m0s
+    version: "4.10"
+  optional: true
+  steps:
+    env:
+      TEMPEST_REGEX: tempest.api.compute.*
```

Comment:
this should be coming form the nova-operator repo not set here.

Reply:
This is just to test which api tests we will execute, after that we will move it to the tempest-skiplist project, don't worry

Reply:
well i dont want this to be in a single project the  way it was done in tripleo
i prefer the jobs that are run on the nova operator are configured form the nova operator repo.